standalone/src/models/product-type/product-type/presets/tshirt.ts

```diff
@@ -0,0 +1,45 @@
+import { type TBuilder } from '@/core';
+import {
+  AttributeDefinitionRest,
+  AttributeDefinitionGraphql,
+  AttributeDefinition,
+} from '../..';
+import {
+  CompatModelBuilder,
+  GraphqlModelBuilder,
+  RestModelBuilder,
+} from '../builders';
+import type {
+  TProductType,
+  TProductTypeGraphql,
+  TProductTypeRest,
+} from '../types';
+
+export const restPreset = (): TBuilder<TProductTypeRest> =>
+  RestModelBuilder()
+    .name('T-shirt Product Type')
+    .attributes([
+      AttributeDefinitionRest.presets.countryOfOrigin().buildRest(),
+      AttributeDefinitionRest.presets.size().buildRest(),
```

Comment:
You don't need to use the `buildRest` method here. That's only needed for the compatibility model.

```suggestion
      AttributeDefinitionRest.presets.countryOfOrigin().build(),
      AttributeDefinitionRest.presets.size().build(),
```

Reply:
Fixed.